Tests/SwiftCrossUITests/SwiftCrossUIBackendTests.swift

```diff
@@ -0,0 +1,158 @@
+import Testing
+
+@testable import SwiftCrossUI
+
+#if canImport(AppKitBackend)
+    @testable import AppKitBackend
+#elseif canImport(DefaultBackend)
+    // Is this close enough to a "Mock Backend"?
+    @testable import DefaultBackend
+#endif
+
+// TODO: Create mock backend so that this can be tested on all platforms. There's
+//       nothing AppKit-specific about it.
+@Suite(
+    "Testing for Graphical Backends",
+    .tags(.Backend)
+)
+struct BackendTests {
+    enum BackendTestError: Error {
+        case FailedBitmapBack
+        case FailedTiffRep
+
+        var errorDescription: String? {
+            switch self {
+                case .FailedBitmapBack: "Failed to create bitmap backing"
+                case .FailedTiffRep: "Failed to create tiff representation"
+            }
+        }
+    }
+
+    #if canImport(AppKitBackend)
+        @Test(
+            "Validates Observation will not give up the thread entirely when sleeping the thread",
+            .tags(.Observation)
+        )
+        func ThrottledStateObservation() async {
```

Comment:
This isn't a backend test, it's a state test, it just happens to require a reference to a backend. In this case, DefaultBackend should work, so feel free to replace the use of AppKitBackend if you want (in just this test, not the real backend test).